src/classes/WalletConnectors/index.ts

```diff
@@ -1,8 +1,24 @@
+import { icons } from '../../assets/icons/icons'
+
+import { MetaMaskConnector } from './MetaMaskConnector'
+import { SafePalConnector } from './SafePalConnector'
+import { BitgetConnector } from './BitgetConnector'
+import { TrustWalletConnector } from './TrustWalletConnector'
+import { CoinbaseConnector } from './CoinbaseConnector'
+
 export * from './WalletConnector'
 export * from './BrowserConnector'
 
-export { BitgetWallet } from './BitgetWallet'
-export { MetaMaskWallet } from './Metamask'
-export { SafePalWallet } from './SafePal'
-export { TrustWallet } from './TrustWallet'
-export { CoinbaseWallet } from './CoinbaseWallet'
+export const metamaskWalletConnector = new MetaMaskConnector({ icon: icons.metamask })
+export const bitgetConnector = new BitgetConnector({ icon: icons.bitkeep })
+export const safePalConnector = new SafePalConnector({ icon: icons.safepal })
+export const trustWalletConnector = new TrustWalletConnector({ icon: icons.trustwallet })
+export const coinbaseConnector = new CoinbaseConnector({ icon: icons.coinbasewallet })
```

Comment:
I think it's better to export the class rather than the object itself, that way we give more freedom to anyone using it to add other functionalities on top.